.github/workflows/test.yml

```
@@ -25,3 +25,14 @@ jobs:
       - name: Test
         run: go test -v ./...
 
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2.5.2
```

Picky, but there's no need to name this.
```suggestion
      - uses: golangci/golangci-lint-action@v2.5.2
```